site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig

```diff
@@ -28,40 +28,38 @@
 		{# text box with the path to the current config file, hidden but still used in js #}
 		<input style='width: 83%' type='hidden' id='input_provide_full_path' name='autograding_config_path' 
 			value="{{ gradeable.getAutogradingConfigPath() }}" class="required" placeholder="(Required)" {{'disabled'}}/>
-    </fieldset>
+    </div>
 
     <div class="btn-container">
         <input type="button" class="btn btn-default" id="show-advanced-student-permissions" value="Show Student Permission Settings" onclick="showAdvancedStudentPermissions()">
         <span id="hide-advanced-student-permissions-container" hidden><input type="button" class="btn btn-primary student_permissions" value="Hide Student Permission Settings" onclick="hideAdvancedStudentPermissions()"></span>
     </div>
-    <div class="student_permissions" hidden>
-        <fieldset>
-            <div>
-                What are students allowed to do?
-            </div>
+    <div class="student_permissions settings" hidden>
```

Comment:
Changing the name of this variable seems to have messed with the js of the "Show Student Permission Settings" button.  Now when the page is initially loaded, the permission settings in the div are shown even though the white "Show" button is displayed (and clicking it for the first time does nothing).  Prior to the changes, the settings are hidden until the button is pressed.

Reply:
I believe I fixed this in my last commit